front/app/modules/commercial/custom_topics/admin/containers/TopicsSettings/New/index.tsx

```diff
@@ -8,68 +8,33 @@ import messages from '../messages';
 // components
 import GoBackButton from 'components/UI/GoBackButton';
 import { Section, SectionTitle } from 'components/admin/Section';
-import { Formik } from 'formik';
 import TopicForm from '../TopicForm';
 
 // services
 import { addTopic, ITopicUpdate } from '../../../../services/topics';
 
-// typings
-import { CLErrorsJSON } from 'typings';
-
-// utils
-import { isCLErrorJSON } from 'utils/errorUtils';
-
-interface Props {}
-
-export default class New extends React.Component<Props> {
-  handleSubmit = (
-    values: ITopicUpdate,
-    { setErrors, setSubmitting, setStatus }
-  ) => {
-    addTopic({
+const New = () => {
+  const handleSubmit = async (values: ITopicUpdate) => {
+    await addTopic({
```

Comment:
So these API errors that were originally handled here are now handled via  `handleHookFormSubmissionError(error, methods.setError);` in the TopicForm/Index file correct?

Reply:
Correct! :)